app/views/shared/_cookies_banner.html.erb

```diff
@@ -6,16 +6,14 @@
           Tell us whether you accept cookies
         </h1>
         <p class="govuk-body">
-        We use <%= govuk_link_to 'cookies to collect information', cookies_path %> about how you use ‘<%= service_name %>’. We use this information to make the website work as well as possible and improve government services.
+          We use <%= govuk_link_to 'cookies to collect information', cookies_path %> about how you use ‘<%= service_name %>’. We use this information to make the website work as well as possible and improve government services.
         </p>
-        <div class="cookies_banner__button govuk-grid-column-full govuk-grid-column-one-half-from-desktop">
-          <%= form_with model: CookiePreferencesForm.new(consent: 'yes'), url: cookie_preferences_path, method: :post do |f| %>
-            <%= f.govuk_submit 'Accept all cookies' %>
-          <% end %>
-        </div>
-        <div class="cookies_banner__button govuk-grid-column-full govuk-grid-column-one-half-from-desktop">
-          <a class="govuk-button" href=<%= cookies_path %>>Set cookie preferences</a>
-        </div>
+        <%= form_with model: CookiePreferencesForm.new(consent: 'yes'), url: cookie_preferences_path, method: :post do |f| %>
+          <div class="govuk-button-group">
+            <%= f.govuk_submit 'Accept all cookies', classes: 'app-button--cookies' %>
+            <a class="govuk-button app-button--cookies" href=<%= cookies_path %>>Set cookie preferences</a>
+          </div>
```

Comment:
👋🏽  hey, I [updated the formbuilder](https://github.com/DFE-Digital/govuk_design_system_formbuilder/pull/237) yesterday to make this easier; you should be able to just do this for the same result.

```erb
<%= f.govuk_submit('Accept all cookies', classes: 'app-button--cookies') do %>
  <a class="govuk-button app-button--cookies" href=<%= cookies_path %>>Set cookie preferences</a>
<% end %>
```

Reply:
Ooooooh, nice one!

Reply:
Done! Also realise can use our `govuk_button_link_to` helper here as well.